File: mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);

    for (int i = height; i > 0; i--)
    {
        for (int j = 1; j <= height; j++)
        {
            if (j >= i)
            {
                printf("#");
            }
            else
            {
                printf(" ");
            }
        }

        printf("  ");

        for (int k = height; k > 0; k--)
        {
            if (k >= i)
            {
                printf("#");
            }
            else
            {
                continue;
            }
        }
        printf("\n");
    }
}

File: sentimental-mario-more/mario.py
# TODO
import cs50

height = 0
while height < 1 or height > 8:
    height = cs50.get_int("Height: ")

for row in range(1, height + 1):
    print(" " * (height - row), end="")
    print(row * "#" + "  " + row * "#")
